src/com/nisovin/magicspells/castmodifiers/conditions/VarStringEqualsCondition.java

```diff
@@ -0,0 +1,67 @@
+package com.nisovin.magicspells.castmodifiers.conditions;
+
+import org.bukkit.Location;
+import org.bukkit.entity.LivingEntity;
+import org.bukkit.entity.Player;
+
+import com.nisovin.magicspells.DebugHandler;
+import com.nisovin.magicspells.MagicSpells;
+import com.nisovin.magicspells.castmodifiers.Condition;
+
+public class VarStringEqualsCondition extends Condition {
+
+	String variable;
+	String value;
+	
+	@Override
+	public boolean setVar(String var) {
+		try {
+			String s = var;
+			int colonposition = var.indexOf(":");
```

Comment:
You might want to consider using the split method provided by the String class, detailed here https://docs.oracle.com/javase/7/docs/api/java/lang/String.html#split(java.lang.String,%20int), it allows you to specify a maximum number of parts to produce. It would save a bit of logic as an alternative to manually splitting based upon substrings.